tests/components/labOrderEntry/__snapshots__/LabEntryForm.test.jsx.snap

```diff
@@ -265,6 +265,7 @@ exports[`Component: LabEntryForm should render on initial setup 1`] = `
         <form>
           <LabPanelFieldSet
             handleTestSelection={[Function]}
+            labCategoryName={null}
```

Comment:
A couple snapshot tests that were failing, assumedly prior to my changes as well, I just updated them to pass.